### PR TITLE
Add mitigation lab

### PR DESCRIPTION
**Pull request by LauraDamianTNA**
This PR adds the mitigation lab with examples for content safety and blocked lists.

### CODE REVIEW

File: guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Guardrail example: Groundedness check
+
+One way of mitigating hallucination by an agent, is to check for each agent's response if it is grounded in the content. One way of doing this is by using Azure Content Safety's Groundedness API.
+
+## Create an Azure Content Safety instance
+
+In the [Azure Portal](https://portal.azure.com), type **Azure Content Safety** in the search bar and create an new instance if needed.
```
Comment (MaggieSalak) [inline guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md:7]:
```suggestion
In the [Azure Portal](https://portal.azure.com), type **Azure Content Safety** in the search bar and create a new instance if needed.
```

**Review by chrislomonico — COMMENTED** [review]
reviewed lab submissions

File: guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+Please use **East US** as the region for deployment. As of today when writing this lab content, this is the only region which has all (preview) features available. You can review the [regional availability for ACS here](https://learn.microsoft.com/en-us/azure/ai-services/content-safety/overview#region-availability).
```
Comment (chrislomonico) [inline guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md:9]: Just checking, is this still true that `East US` is on the only region?

Comment (LauraDamianTNA) [reply guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md:9]: Yes, according to this documentation: https://learn.microsoft.com/en-us/azure/ai-services/content-safety/overview#region-availability

File: guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+# Guardrail example: Groundedness check
```
Comment (MaggieSalak) [inline guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md:1]: Is this file linked somewhere? I couldn't find it linked in the original lab and just wondering how it fits in the lab.

Comment (MaggieSalak) [reply guardrail-lab/03-Mitigate/examples/hallucination-groundedness.md:1]: I found that the whole `examples` folder is linked from the readme.

**Review by MaggieSalak — APPROVED** [review]

**Pull request merged**